whois-commons/src/main/java/net/ripe/db/whois/common/WhoisRipePropertyResolver.java

```diff
@@ -16,6 +22,28 @@
 @Profile({WhoisProfile.RIPE_DEPLOYED, WhoisProfile.TEST})
 public class WhoisRipePropertyResolver {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(WhoisRipePropertyResolver.class);
+
+    static {
+        String hostName = System.getenv("HOSTNAME");
+        if (StringUtils.isBlank(hostName)) {
+            try {
+                hostName = InetAddress.getLocalHost().getHostName();
+            } catch (UnknownHostException ignored) {
+                LOGGER.debug("{}: {}", ignored.getClass().getName(), ignored.getMessage());
+            }
+
+            if (StringUtils.isBlank(hostName)) {
+                System.err.println("Acquiring HOSTNAME failed! Try\nexport HOSTNAME=$(hostname -s)\n");
```

Comment:
Use `Logger.error() `not `System.err.println()`

Reply:
Yes, I just copy pasted the old the code. I will correct this one